lambdas/matching/src/matching-handler.ts

```diff
@@ -40,10 +40,21 @@ export class MatchingHandler implements LambdaInterface {
       const txn = response.headers.get("x-amz-cf-id") ?? "";
       addLogEntry(event, txn, context);
       const contentType = response.headers.get("content-type");
+
       if (contentType?.includes("application/json")) {
+        let responseJson;
+        try {
+          responseJson = await response.json();
+        } catch (error) {
+          // Sometimes we receive text bodies for the application/json content-type
```

Comment:
```suggestion
```
I don't believe this comment is necessary (and we should avoid them in most cases)

Reply:
I did it because somebody new without context is gonna think "why the heck are we doing this"?

Reply:
I think this line of code explains it
`if (contentType?.includes("application/json")) `